src/ispd/motor/queueNetworks/CentrosDeServico.java

```diff
@@ -453,7 +453,7 @@ public NoFila peekFilaMestre() {
     }
 
     public NoFila peekFilaComun() {
-        System.out.printf("getTipo() = %d \n", getTipo());
+        System.out.printf("getType() = %d \n", getTipo());
```

Comment:
Revert this, must've done by accident when changing the name of the method in the workload generator interface.